perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_sshPublicKey.java

```diff
@@ -17,18 +25,124 @@
  */
 public class urn_perun_user_attribute_def_def_sshPublicKey extends UserAttributesModuleAbstract implements UserAttributesModuleImplApi {
 
+	private byte[] bytes = null;
+	private int pos = 0;
```

Comment:
I think these fields should be moved to methods as local variables because an instance of this class is actually shared between threads/requests and it wouldn't be thread-safe with these fields.